### ISSUE TITLE
Input boxes and text are confusing on login/signup screens

### ISSUE THREAD

**Issue by roman87** [TeamMentor/TM_4_0_Design#409]
I think we had this issue once before. The login boxes are no longer boxes (at least on my screen) and are instead represented by a hard to see line. 
Text like "USERNAME" now blends in with the helper text of "Username"  and with instruction text below it. 
As I said, I, personally,  liked the previous login/signup page better. 

What we need to get in the habit of, is to make these changes in the POC. Especially after beta2 comes out. 

![selection_092](https://cloud.githubusercontent.com/assets/1326120/6101322/2d99f9e0-aff0-11e4-9f14-6726bce3685a.png)
![selection_093](https://cloud.githubusercontent.com/assets/1326120/6101323/2f853fc6-aff0-11e4-8d71-cb832fdf5143.png)

**Comment by michaelhidalgo** [comment]
I also noticed the border in the text box is missing. I gave it a test on safari and Chrome in the mac and the image looks like the images included above.

**Comment by luchia** [comment]
Sure, I have changed it back but I did need to come up with a different variation. I did some reading last week and box-shadow css property is not visible in all browsers (IE 8 and below). That is why I need to change it.

I have fixed this in the referenced commit.

![screen shot 2015-02-10 at 7 47 59 am](https://cloud.githubusercontent.com/assets/4547737/6116561/6fc03aa0-b0f9-11e4-978f-7a0b7ec89614.png)
![screen shot 2015-02-10 at 7 48 16 am](https://cloud.githubusercontent.com/assets/4547737/6116560/6fbefe1a-b0f9-11e4-9c83-4de0c98598f4.png)
